src/asn1.js

```diff
@@ -2825,7 +2825,22 @@ class LocalSidValueBlock extends HexBlock(LocalBaseBlock)
 
 		return (inputOffset + this.blockLength);
 	}
-	//**********************************************************************************
+//**********************************************************************************
+	/**
+   * Save a BigInt value immediately as an array of octects.
+   */
+ set valueBigInt(value) {
```

Comment:
Let's add an assert function like
```js
function assertBigInt() {
  if (typeof BigInt === "undefined") {
    throw new Error("BigInt is not defined. Your environment doesn't implement BigInt.")
  }
}
```

and use it here

```suggestion
	set valueBigInt(value)
	{
		assertBigInt();
```

Reply:
And please format your changes. Use TAB instead of SPACE for padding